platform/android/app/src/main/kotlin/coop/polypoly/polypod/PodApiError.kt

```diff
@@ -0,0 +1,13 @@
+package coop.polypoly.polypod
+
+
+interface PodApiErrorInterface {
```

Comment:
This seems a bit weird - while I'm OK with it (we can always refactor later), I think it could be a lot simpler:

```
class PodApiError : Exception {
    static fun endpointError(fetchType: String) = PodApiError("endpoint.$fetchType failed")
}
```

I don't see why we'd specifically need an interface, why we'd need an instance of `PodApiError` - even if we did, why the class is anonymous... Just a bit weird :D But like I said, not crucial.